zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -1423,21 +1423,99 @@ paths:
         "500":
           $ref: "#/components/responses/InternalServerError"
 
+  /variables:
+    post:
+      tags:
+        - Variable
+      operationId: createVariable
+      summary: Create variable
+      description: |
+        Create cluster level variables
```

🔧 We're just creating one variable here.

```suggestion
        Create a cluster-level variable
```

---

Corrected